pkgs/top-level/all-packages.nix

```
@@ -22946,6 +22946,12 @@ with pkgs;
 
   toml-f = callPackage ../development/libraries/toml-f { };
 
+  fypp = callPackage ../development/python-modules/fypp {
```

Just grepping around for `buildPythonApplication` it yields two patterns that are commonly used:
```suggestion
  fypp =python3Packages.callPackage  ../development/python-modules/fypp {
```
and using `python3Packages` as input (i.e. using then `python3Packages.buildPythonApplication` directly).